docs/docs/more-info/architecture.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: architecture
+title: Architecture
+sidebar_label: Architecture
+keywords:
+  - Instant OpenHIE
+  - Architecture
+description: A description of the instant OpenHIE technical architecture
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::info
+The Instant OpenHIE architecture, codebase, and documentation are under active development and are subject to change. While we encourage adoption and extension of the Instant OpenHIE framework, we do not consider this ready for production use at this stage.
+:::
+
+The fundamental concept of Instant OpenHIE is that it can be extended to support additional use cases and workflows. This is achieved through packages. Packages conforms to a specific set of rules and contain scripts and resources that allow particular applications to be spun up and configured on the platforms supported by Instant OpenHIE. The currently supported platforms are **local deployments via Docker and Docker Compose** and deployment to a **Kubernetes cluster**.
```

Comment:
Typo
```suggestion
The fundamental concept of Instant OpenHIE is that it can be extended to support additional use cases and workflows. This is achieved through packages. Packages conform to a specific set of rules and contain scripts and resources that allow particular applications to be spun up and configured on the platforms supported by Instant OpenHIE. The currently supported platforms are **local deployments via Docker and Docker Compose** and deployment to a **Kubernetes cluster**.
```